e2e-test/src/test/java/org/alfresco/hxi_connector/e2e_test/util/TestJsonUtils.java

```diff
@@ -0,0 +1,49 @@
+/*-
+ * #%L
+ * Alfresco HX Insight Connector
+ * %%
+ * Copyright (C) 2023 - 2024 Alfresco Software Limited
+ * %%
+ * This file is part of the Alfresco software.
+ * If the software was purchased under a paid Alfresco license, the terms of
+ * the paid license agreement will prevail.  Otherwise, the software is
+ * provided under the following open source license terms:
+ *
+ * Alfresco is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * Alfresco is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with Alfresco. If not, see <http://www.gnu.org/licenses/>.
+ * #L%
+ */
+package org.alfresco.hxi_connector.e2e_test.util;
+
+import static lombok.AccessLevel.PRIVATE;
+
+import java.util.Set;
+
+import com.fasterxml.jackson.core.type.TypeReference;
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import lombok.NoArgsConstructor;
+import lombok.SneakyThrows;
+
+@NoArgsConstructor(access = PRIVATE)
+@SuppressWarnings({"PMD.TestClassWithoutTestCases", "PMD.FieldNamingConventions"})
+public final class TestJsonUtils
+{
+    private static final ObjectMapper objectMapper = new ObjectMapper();
+
+    @SneakyThrows
+    public static Set<String> getSetProperty(JsonNode jsonNode, String propertyName)
```

Comment:
Method name is really confusing 😅 What this method does? 🧐

Reply:
it returns property of type set😂 maybe I should just create "toSet(String)" method instead?

Reply:
This sounds better, I'm for 🙋‍♂️